tests/test_auth.py

```diff
@@ -53,12 +62,35 @@ def test_get_token(auth_mock):
     assert auth_mock.token == TOKEN
 
 
+def test_repr_project():
```

Comment:
None of these are needed, auth an internally used classed, should not be printed anywhere anyway